test/position-managers/PositionManager.modifyLiquidities.t.sol

```diff
@@ -837,6 +837,16 @@ contract PositionManagerModifyLiquiditiesTest is Test, PosmTestSetup, LiquidityF
         assertEq(lpm.getPositionLiquidity(tokenId), initialLiquidity + newLiquidity);
     }
 
+    struct BalanceDiff {
+        uint256 before;
+        uint256 _after;
```

Comment:
is the underscore needed? would prefer `.after` and `.0` and `.1`

Reply:
after is a reserved keyword, also you cannot use pure numbers as variable names

Reply:
sad